packages/teams-js/src/public/authentication.ts

```diff
@@ -202,6 +195,12 @@ export namespace authentication {
     });
   }
 
+  function throwIfStringNotValidHttpsUrl(link: string): void {
```

Comment:
I see that in Niharika's PR she's also adding URL validation that is slightly different than this one. Is it possible to reconcile them?

https://github.com/OfficeDev/microsoft-teams-library-js/pull/2272/files#diff-6deac44c9539303a6e3b3d416f349dc7109be94374522c76e78161acd98ece25

Reply:
Yes, we can reconcile them. It seems like her PR should either be updaing the existing `isValidHttpsUrl` function, or it should be introducing a new function that throws errors instead of returning a boolean, but share the underlying "is this URL valid" logic with the logic in `isValidHttpsURL`

Reply:
I will wait for Niharika to check in and then use her new `validateUrl` function for consistency.